concrete/src/Search/Pagination/PermissionablePagination.php

```diff
@@ -22,7 +22,7 @@ public function __construct(ItemList $itemList)
         $adapter = new ArrayAdapter($results);
         $this->list = $itemList;
```

Comment:
We can also remove this line, can't we?

Reply:
You're right, good catch, that also gets set in the parent. I've removed that now.